R/app_ui.R

```diff
@@ -80,17 +55,38 @@ app_ui <- function(request) {
 #' @importFrom golem add_resource_path activate_js favicon bundle_resources
 #' @noRd
 golem_add_external_resources <- function() {
-  add_resource_path(
-    "www", app_sys("app/www")
-  )
+  # add_resource_path(
+  #   "www", app_sys("app/www")
+  # )
+  
+  resources <- app_sys("app/www")
+  addResourcePath("www", resources)
 
   tags$head(
-    favicon(),
-    bundle_resources(
-      path = app_sys("app/www"),
-      app_title = "nhsbsaShinyR"
-    )
+    favicon("assets/favicons/favicon"),
+    tags$title("nhsbsaShinyR"),
+    # bundle_resources(
+    #   path = "www",
+    #   app_title = "nhsbsaShinyR"
+    # ),
     # Add here other external resources
     # for example, you can add shinyalert::useShinyalert()
+    
+    # Javascript resources
+    htmltools::htmlDependency(
+      name = "resources",
+      version = "0.0.1",
+      src = resources,
+      script = list.files(resources, pattern = "\\.js$", recursive = TRUE),
+      package = NULL,
+      all_files = TRUE
+    ),
+    
+    # CSS resources
+    lapply(
+      list.files(resources, pattern = "\\.css$", recursive = TRUE),
+      function(x) tags$link(href = file.path("www", x), rel = "stylesheet")
+    )
+    
```

Comment:
Some modifications to load in all css and js files in all sub-folders of www. This allows better organisation of static files.